src/pretix/plugins/sendmail/views.py

```diff
@@ -29,6 +29,12 @@ class SenderView(EventPermissionRequiredMixin, FormView):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['event'] = self.request.event
+        from_log_id = self.request.GET.get('from_log')
+        message = LogEntry.objects.get(id=from_log_id).display()
```

Comment:
Hi @asv-hungvt, I'm unsure if I really understand your question correctly. Something like the following should work:

```
if 'from_log' in self.request.GET:
    try:
        message = LogEntry.objects.get(
            id=from_log_id, 
            event=self.request.event,
            action_type='pretix.plugins.sendmail.sent'
        ).display()
    except LogEntry.DoesNotExist:
        raise Http404(_('You supplied an invalid log entry ID'))
```